docs/source/usage/wm-widgets.rst

```diff
@@ -51,7 +51,7 @@ Add the following to :file:`~/.config/qtile/config.py`:
 
 .. _bar-usage:
 
-LemonBoy’s bar
+bar-aint-recursive
 ==============
```

Comment:
You must adjust the number of `=` signs. This is not a nitpicking: Sphinx *requires* header underlines to be wider or have equal width then the header.